### DISCUSSION TITLE
access to trello

### DISCUSSION THREAD

**Discussion by Javican** [FireMarshmellow/Spotlight_Storage#54] [Ideas]
How do we create ideas on trello, do we add them directly or should we go trough a special process?

**Comment by FireMarshmellow** [comment]
Everyone should have permission to common, And edit the bords.

**Reply by simen64** [reply]
Hi, where should we contact to get access to the Trello board now?

**Reply by Prankhouz** [reply]
Yeah it appears the board has been closed.
